From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Samuel Maddock <smaddock@slack-corp.com>
Date: Fri, 28 Mar 2025 16:57:23 -0400
Subject: fix: install new cppgc header files

https://chromium-review.googlesource.com/c/v8/v8/+/6348644
Copies new cppgc/internal header file required for building electron_node headers.

This updates the files initially added in an upstream Node PR.
https://github.com/nodejs/node/pull/50737

diff --git a/tools/install.py b/tools/install.py
index 7232be863d517c8445059a57c5938c2cbdeaf81b..92f83a83be67aafc9ead6923b868dbb0de39db34 100755
--- a/tools/install.py
+++ b/tools/install.py
@@ -212,6 +212,7 @@ def headers(options, action):
       'include/cppgc/internal/caged-heap-local-data.h',
       'include/cppgc/internal/caged-heap.h',
       'include/cppgc/internal/compiler-specific.h',
+      'include/cppgc/internal/conditional-stack-allocated.h',
       'include/cppgc/internal/finalizer-trait.h',
       'include/cppgc/internal/gc-info.h',
       'include/cppgc/internal/logging.h',
